packages/help-center/src/components/help-center-contact-page.tsx

```diff
@@ -131,9 +131,9 @@ export const HelpCenterContactPage: FC = () => {
 				{ supportActivity && <HelpCenterActiveTicketNotice tickets={ supportActivity } /> }
 				{ /* Easter */ }
```

Comment:
Probably good to remove this comment :) I don't think we need it at all. If needed, `git blame` provides the correct and always up to date context for these dates.

Reply:
I agree! Thank you!